dockers/base-ipu/Dockerfile

```diff
@@ -0,0 +1,97 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:20.04
```

Comment:
what is the difference between the `base-conda`?

Reply:
This one doesn't include nvidia components - ideally we'd base this on the graphcore/poplar image, but that's not public currently

Reply:
so you say that only change is the base image, so shall we make it rather as another argument?

Reply:
it also doesn't install apex, DALI etc

Reply:
good point but, then it is rather about having CPU and GPU conda image...